pkg/kafka/auth/options.go

```diff
@@ -49,6 +49,16 @@ const (
 
 	defaultPlainTextUserName = ""
 	defaultPlainTextPassword = ""
+
+	// Scram configuration options
+	scramPrefix          = ".scram"
+	suffixScramUserName  = ".username"
+	suffixScramAlgorithm = ".algorithm"
+	suffixScramPassword  = ".password"
+
+	defaultScramUserName  = ""
+	defaultScramPassword  = ""
+	defaultScramAlgorithm = ""
```

Comment:
```suggestion
	defaultScramAlgorithm = "sha512"
```
unless you want to set this to a default value, this variable is not needed.

Reply:
Set to default sha512